SecurityPkg/Library/BaseHash2CryptoLibNull/BaseHash2CryptoLibNull.uni

```diff
@@ -0,0 +1,13 @@
+// /** @file
+// Null instance of Hash 2 Crypto Library.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+// **/
+
+// MU_CHANGE - NEW FILE
+
+#string STR_MODULE_ABSTRACT             #language en-US "Null Instance of Hash 2 Crypto Library"
+
+#string STR_MODULE_DESCRIPTION          #language en-US "Null instance of Hash 2 Crypto Library."

```

Comment:
I'm fine with this, but we seem inconsistent about whether we have uni files or not for files that don't actually have strings

Reply:
I usually do include them in "core" code, often they have not been. I'll keep this file.